geode-tcp-server/src/test/java/org/apache/geode/distributed/internal/tcpserver/TcpServerJUnitTest.java

```diff
@@ -344,19 +267,29 @@ public void endRequest(Object request, long startTime) {}
     public void endResponse(Object request, long startTime) {}
   }
 
-  private/* GemStoneAddition */ static class SimpleStats implements PoolStatHelper {
-    AtomicInteger ended = new AtomicInteger();
-    AtomicInteger started = new AtomicInteger();
 
+  public class InfoRequestHandler implements TcpHandler {
```

Comment:
why was `org.apache.geode.distributed.internal.InfoRequestHandler` no longer suitable?

Reply:
That's a geode-core class.  It gets information from GemFireVersion, also in geode-core.

Reply:
bravo!